src/components/side_panel/chart/building_blocks/chart_title/chart_title.ts

```diff
@@ -34,6 +36,7 @@ export class ChartTitle extends Component<Props, SpreadsheetChildEnv> {
   };
   static defaultProps = {
     title: "",
+    placeholder: "",
```

Comment:
To keep consistency with the previous behaviour, should we add a default props `Title` for `name` ?
Or even better, make `name` props required.